src/components/Container/Container.spec.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import assert from 'assert';
+import { common } from '../../util/generic-tests';
+
+import Container from './Container';
+
+describe('Container', () => {
+	common(Container);
+
+	describe('Header', () => {
+		it('renders a header element when included.', () => {
+			const wrapper = shallow(<Container><Container.Header>Header</Container.Header><Container.Content>Content</Container.Content></Container>);
```

Comment:
Could you wrap these lines better, here's [an example from dropmenu](https://github.com/appnexus/lucid/blob/master/src/components/DropMenu/DropMenu.spec.jsx#L41-L48)